src/machine/board_wioterminal.go

```diff
@@ -0,0 +1,160 @@
+// +build sam,atsamd51,wioterminal
+
+package machine
+
+import "device/sam"
+
+// used to reset into bootloader
+const RESET_MAGIC_VALUE = 0xf01669ef
+
+// GPIO Pins
+const (
+	D0  = PB17 // UART0 RX/PWM available
+	D1  = PB16 // UART0 TX/PWM available
+	D4  = PA14 // PWM available
+	D5  = PA16 // PWM available
+	D6  = PA18 // PWM available
+	D8  = PB03 // built-in neopixel
+	D9  = PA19 // PWM available
+	D10 = PA20 // can be used for PWM or UART1 TX
+	D11 = PA21 // can be used for PWM or UART1 RX
+	D12 = PA22 // PWM available
+	D13 = PA23 // PWM available
+	D21 = PA13 // PWM available
+	D22 = PA12 // PWM available
+	D23 = PB22 // PWM available
+	D24 = PB23 // PWM available
+	D25 = PA17 // PWM available
```

Comment:
I could not easily find these in the schematics. Are these pins available on the WioTerminal?

Reply:
I haven't made it to the correct value yet.

The schematic can be found below.
https://files.seeedstudio.com/wiki/Wio-Terminal/res/Wio-Terminal-Schematics.pdf

Is the definition of Dxx correct to set D0 to D8 in the Pinout Diagram?
Is it right to fit the Arduino?

Eventually, I'm going to adjust to the following.

https://github.com/Seeed-Studio/ArduinoCore-samd/blob/master/variants/wio_terminal/variant.h
https://github.com/Seeed-Studio/ArduinoCore-samd/blob/master/variants/wio_terminal/variant.cpp

For example.

```c
#define D0 (0ul)
const PinDescription g_APinDescription[0] =
        // 0..8 -  Raspberry pi Digital & AD pins
        {PORTB, 8, PIO_ANALOG, (PIN_ATTR_ANALOG | PIN_ATTR_PWM_E), ADC_Channel2, TC4_CH0, TC4_CH0, EXTERNAL_INT_8}, // 0/RPI_D0/RPI_A0
        }
```

I'm going to set it as follows.

```
D0  = PB08
```